### PR TITLE
Docs - list the current configuration options

### PR DESCRIPTION
**Pull request by vinaypyati**
Adding documentation to list the current configuration options of an running `arangod` instance - github issue #6558
Questions:
- Can this be the right place ?
- Backport to previous versions ?

### CODE REVIEW

File: Documentation/Books/Manual/Programs/Arangod/Server.md [hunk 1]
```diff
@@ -269,3 +269,17 @@ value too low can easily overwhelm the server, while setting the value too high
 may result in high memory usage and periodic slowdowns. Value is given in
 microseconds, with a typical range of 100000 (100ms) to 10000000 (10s) and a
 default of 1000000 (1s). Use caution when changing from the default.
+
+## Get current configuration options
+
+To list the configuration options of a running `arangod` instance, you can 
+connect with an ArangoShell and invoke a transaction providing a description
```
Comment (dothebart) [inline Documentation/Books/Manual/Programs/Arangod/Server.md]: link to the arangosh chapter please; link to transaction description

File: Documentation/Books/Manual/Programs/Arangod/Server.md [hunk 2]
```diff
@@ -269,3 +269,17 @@ value too low can easily overwhelm the server, while setting the value too high
 may result in high memory usage and periodic slowdowns. Value is given in
 microseconds, with a typical range of 100000 (100ms) to 10000000 (10s) and a
 default of 1000000 (1s). Use caution when changing from the default.
+
+## Get current configuration options
```
Comment (dothebart) [inline Documentation/Books/Manual/Programs/Arangod/Server.md]: Fetch current

**Comment by sleto-it** [comment]
Thanks for this PR

I think the right place to mention this is in the page

- Manual/Programs/Arangod/Options.md, before "Global Options"

A link to this new subsection could also be added in Administration/Configuration.md

I think having it in Arangod/Server.md hides this important (general) info and is a bit off-topic there as that page is about specific options

@Simran-B WDTY ?

**Comment by sleto-it** [comment]
@vinaypyati regarding your question about backporting: it has to be backported to the 3.4 branch when the final version of this PR is merged into devel

**Pull request merged**